source/transactions/tests/README.rst

```diff
@@ -162,6 +168,13 @@ Each YAML file has the following keys:
 
       - ``errorLabelsOmit``: A list of error label strings that the
         error is expected not to have.
+        
+      - ``state``: A map of expected session states. The keys can be
+        "session0" or "session1", and the values can be any of the states
+        defined in the transactions spec: "starting transaction",
+        "transaction in progress", "transaction comitted", "transaction aborted".
```

Comment:
The Transactions spec doesn't define these states beyond enumerating them. If we end up using a more descriptive name than "state" for this map, then perhaps we can also use shorter strings (e.g. "starting", "in_progress").

Reply:
Happy to change to shorter strings. As you point out these states are not defined as constants, hence what to name them was not entirely clear. I went with the long phrases used in the spec as the starting point.

Reply:
Shorter strings works for me. If we use "transaction" in the spec test field name (as I suggested in https://github.com/mongodb/specifications/pull/548/files#r285253695), we can omit the "transaction" prefix from each state string.